configs/localdocker/node0/config.toml

```diff
@@ -3,3 +3,7 @@ NotaryGroupConfig = "../localdocker.toml"
 [PrivateKeySet]
 SignKeyHex = "0x4344e3372f1b790016ed86400611fab64d0bba91136518685a7ce34106f1c759"
 DestKeyHex = "0xc09363e85290020c132053f029f23988d4e35c521b0698312e0071f3e5d0e023"
+
+[Storage]
```

Comment:
I'm now looking at this more and realizing that I'm not sure we separate the tupelo blockstore from the p2p storage needs (which should be in memory). We did that over in community, but not sure that carried over here.

https://github.com/quorumcontrol/community/commit/e30e13fcd86b89e0087d1481a265e3448e80df69

We did that because we saw *significant* slowdown using a non-mem storage for the p2p stuff

Reply:
And could explain why s3 was so bad

Reply:
nodebuilder doesn't use p2p store - it only sets the WithBlockstore so this is totally ok